scanpy/plotting/_stacked_violin.py

```diff
@@ -131,6 +138,14 @@ class StackedViolin(BasePlot):
     # None will draw unadorned violins.
     DEFAULT_INNER = None
 
+    def __getattribute__(self, name: str) -> object:
+        """Called unconditionally when accessing an instance attribute"""
+        if name == "DEFAULT_DENSITY_NORM" and hasattr(self, "DEFAULT_SCALE"):
```

Comment:
```suggestion
        if (name == "DEFAULT_DENSITY_NORM" and hasattr(self, "DEFAULT_SCALE")) or name == 'DEFAULT_SCALE':
```

Is there any scenario in which people are using this publicly? May want to warn on that as well.

Reply:
The problem is that this is a class attribute. So I can’t catch this when it’s set (that would involve a `property` on a metaclass, which is super out of the way and quite complicated)

I instead settled on the warning being thrown when the new attribute is read (which we always do from the instance) while the user has manually set the old one.